src/task/MonitorTask.ts

```diff
@@ -262,6 +262,15 @@ export default class MonitorTask
   }
 
   connectionHealthDidChange(connectionHealthData: ConnectionHealthData): void {
+    if (!this.context.meetingSessionConfiguration?.urls?.audioHostURL?.length) {
```

Comment:
…

Reply:
reverted this line. It did not seem necessary given my testing today.